services/minespace-web/src/components/pages/Project/ProjectSummaryPage.tsx

```diff
@@ -268,6 +268,10 @@ export const ProjectSummaryPage: FC<ProjectSummaryPageProps> = (props) => {
     const newActiveTab = projectFormTabs[currentTabIndex + 1];
     const message = "Successfully saved a draft project description.";
     const values = { ...formValues, status_code: "DFT" };
+    const file_to_upload =
+      values.authorizations.AIR_EMISSIONS_DISCHARGE_PERMIT.AMENDMENT[0].amendment_documents;
+    const filtered = file_to_upload.filter((doc) => !doc.mine_document_guid);
+    values.authorizations.AIR_EMISSIONS_DISCHARGE_PERMIT.AMENDMENT[0].amendment_documents = filtered;
```

Comment:
I don't think mutating the form state like this will work since you're reassigning a form value directly.  For that you can use the redux-form `change` function.  You'd need to add it to the mapDispatchToProps for this file and then this line could like more like this:

```props.change(
      FORM.ADD_EDIT_PROJECT_SUMMARY,
      values.authorizations.AIR_EMISSIONS_DISCHARGE_PERMIT.AMENDMENT[0].amendment_documents,
      filtered
    );```

Reply:
Thank you. Updated as suggested.